masterlist.yaml

```
@@ -13727,8 +13727,13 @@ plugins:
         itm: 4
         udr: 4
 
+  - name: '(AFK_Weye.*|RTT-Weye Relocation Patch|Rumare-AFK_Weye Patch)\.esp'
```

The regex can be simplified a bit:

```suggestion
  - name: '(.*AFK_Weye.*|RTT-Weye Relocation Patch)\.esp'
```